src/backend/api/reviews.js

```diff
@@ -0,0 +1,117 @@
+const express = require("express");
+const { resolve } = require("path");
+const router = express.Router();
+const knex = require("../database");
+
+// 1./api/reviews	GET	Returns all reviews.
+router.get("/", async (request, response) => {
+  try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+    const title = await knex("Review").select("*");
+    if(title.length !== 0){
+      response.send(title);
+    }else {
+      response.status(404).json({Message:"No Reviews Found"});
```

Comment:
I don't think a 404 response is needed here.

Reply:
replaced with 400